compassion_dashboard/README.rst

```diff
@@ -0,0 +1,27 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
I took this header from other modules, it just adds this image to the module description, do you think I should delete it ?